virtual_machine.go

```diff
@@ -643,3 +643,20 @@ func (v *VirtualMachine) ConvertToTemplate(ctx context.Context) (task *Task, err
 	}
 	return NewTask(upid, v.client), nil
 }
+
+func (v *VirtualMachine) UnmountCloudInitISO(ctx context.Context, device string) error {
+	if v.HasTag(MakeTag(TagCloudInit)) {
```

Comment:
nit: turn this into a not check and just return nil and pull the rest of this func out of the if clause
```
if !v.HasTag(MakeTag(TagCloudInit)) {
    return nil 
}
// rest of the func below
```

Reply:
good point